Mathlib/Tactic/Widget/Calc.lean

```diff
@@ -124,19 +126,30 @@ def CalcPanel : Component CalcParams :=
   mk_rpc_widget% CalcPanel.rpc
 
 namespace Lean.Elab.Tactic
-open Meta
+
+open Lean Meta Tactic TryThis in
+/-- Create a `calc` proof. -/
+elab stx:"calc?" : tactic => do
```

Comment:
Can you add a test for this with `guard_msgs`?

Reply:
Done.